vendor/bat-native-ledger/src/bat/ledger/transactions_info.cc

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/transactions_info.h"
+
+#include "base/json/json_reader.h"
```

Comment:
per the comment on `//base` these are ok because they only in the cc file and won't be exposed in the public headers cc @NejcZdovc @ryanml